doc/source/monitoring.rst

```diff
@@ -0,0 +1,25 @@
+Monitoring
+===========
+
+There are mainly three different things:
+ * Rucio internal monitoring using Graphite/Grafana
+ * Transfer monitoring using the messages sent by Hermes
+ * File/Dataset Access monitoring using the traces
+
+
+
+Internal Monitoring:
+--------------------
+
+This is to monitor the internals of Rucio server and daemons, e.g., submission rate of the conveyor, state of conveyor queues, reaper deletion rate, server response times, server active session, etc. We use Graphite[1] for this. It’s easy to setup and then you just have to point your Rucio instance to the Graphite server using the “carbon_server” options in the “monitor” section in etc/rucio.cfg. The different Rucio components then will send metrics using those “record_*” functions you will find all over the code. Graphite has a built-in web interface to show graphs but more comfortable to use is Grafana. You can find an example dashboard from us here [2].
```

Comment:
- server response times -> server response time
- server active session -> server active sessions
- I would remove sentences like "It's easy to setup"...

Reply:
for reference: original text is from Thomas' mail